resources/js/scores-show/info.tsx

```diff
@@ -27,7 +27,7 @@ export default function Info({ score }: Props) {
       </div>
 
       <div className='score-info__item score-info__item--dial'>
-        <Dial accuracy={accuracy(score)} mode={rulesetName(score.ruleset_id)} rank={rank(score)} />
+        <Dial accuracy={accuracy(score)} mode={rulesetName(score.ruleset_id)} rank={rank(score)} rank_cutoffs={rankCutoffs(score)} />
```

Comment:
mode can be removed, also it should be `rankCutoffs` as props that's not json are camelCased